src/data/list/rotate.lean

```diff
@@ -151,6 +152,9 @@ begin
     { simp [rotate_cons_succ, hn] } }
 end
 
+@[simp] lemma nil_eq_rotate_iff {l : list α} {n : ℕ} : [] = l.rotate n ↔ l = [] :=
```

Comment:
For symmetry I think it might make more sense to make this
```suggestion
@[simp] lemma nil_eq_rotate_iff {l : list α} {n : ℕ} : [] = l.rotate n ↔ [] = l :=
```